src/mount/special_write.cc

```diff
@@ -97,6 +97,19 @@ static BytesWritten write(const Context &ctx, const char *buf, size_t size,
 }
 } // InodeTweaks
 
+namespace InodePathByInode {
+static BytesWritten write(const Context &ctx, const char */*buf*/, size_t size,
+	                           off_t off, FileInfo */*fi*/) {
```

Comment:
Please use a space after the asterisk and before the commented arguments to avoid strange formatting in some editors or use [[maybe_unused]].

```cpp
static BytesWritten write(const Context &ctx, const char * /*buf*/, size_t size,
	                           off_t off, FileInfo * /*fi*/) {
```

Also, please fix the indentation.

Reply:
Done